packages/common/src/hooks/useCoinflowAdapter.ts

```diff
@@ -46,6 +47,7 @@ export const useCoinflowWithdrawalAdapter = () => {
   } = useAppContext()
   const [adapter, setAdapter] = useState<CoinflowAdapter | null>(null)
   const feePayerOverride = useSelector(getFeePayer)
+  const { currentUser } = useSelector(getWalletAddresses)
```

Comment:
I think this is a case where we actually want to use the `web3User` and not the `currentUser`, since this is going to involve signing a transaction to move funds from a user bank. You can't do that on behalf of a managed account (you can't do any token operations in manager mode).

Or I guess we could just leave this as a pattern and otherwise disable the code path in manager mode, so you don't end up here.

Reply:
Actually, maybe ignore me. I think this would get super confusing to a user if they were in manager mode and some kind of UI glitch caused it to move tokens out of the manager's account. In general, these things just shouldn't work if you're trying to do it while in manager mode.

Reply:
wait yeah.. using web3User would mean the manager's funds would be transferred right?

Not sure if there was a conversation around disabling certain features in manager mode.. tho I can see a case where we'd want the manager to handle withdrawals for the artist? not sure